src/main/java/com/aiykr/iquais/controller/UserController.java

```diff
@@ -24,7 +27,7 @@ public ResponseEntity<Response<List<UserResponseDTO>>> getAllUsers() {
 		return userService.getAllUsers();
 	}
 	@PostMapping("/users/student")
-	public ResponseEntity<Response> createStudent(@RequestBody PostUserDTO postUserDTO) throws ResponseExceptionHandler {
+	public ResponseEntity<Response> createStudent(@RequestBody PostUserDTO postUserDTO) throws IquaisException {
 		return userService.createUser(postUserDTO);
```

Comment:
Should the service be directly creating a Response Entity? shouldn't it be the controllers job

Reply:
changed as per suggestion